src/safeposix/syscalls/sys_calls.rs

```diff
@@ -94,19 +128,26 @@ impl Cage {
                     }
                 }
             } else {
+                // If the mutex is `None` - we mimic the same behavior in the child's mutext table
                 new_mutex_table.push(None);
             }
         }
         drop(mutextable);
 
-        //construct a new condvar in the child cage where each initialized condvar is in the parent cage
+        // Construct a replica of the condition variables table in the child cage object
```

Comment:
A comment about what "Condition variables table" store would be great for understanding the context.